package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "eslint src src-clean .storybook",
     "dev": "concurrently --kill-others \"nodemon --watch src\" \"node webpack/webpack-dev-server\"",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build:clean": "del \"dist/!(.git*|Procfile)**\"",
+    "build:clean": "del-cli \"dist/!(.git*|Procfile)**\"",
```

Comment:
`del` is reserved command in WIndows. `del-cli` is just alias for `del` package. Does it work on Mac and Linux?